### ISSUE TITLE
No Default Save

### ISSUE THREAD

**Issue by Eddie-Hartman** [MJWherry/SQLScriptGenerator#2]
I didn't do a lot of testing, but it seems that the save button does not work if you haven't saved before. I was only able to save using the save as button.

**Comment by MJWherry** [comment]
Yes i know. I havent worked on that really since summer since i h ve like 3 other projects to do. 


Sent from my Sprint Samsung Galaxy S® 6.-------- Original message --------From: "Elton (Eddie) Hartman" <notifications@github.com> Date: 9/13/2016  11:44 PM  (GMT-05:00) To: MJWherry/SQLScriptGenerator <SQLScriptGenerator@noreply.github.com> Subject: [MJWherry/SQLScriptGenerator] No Default Save (#2) 
I didn't do a lot of testing, but it seems that the save button does not work if you haven't saved before. I was only able to save using the save as button.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/MJWherry/SQLScriptGenerator","title":"MJWherry/SQLScriptGenerator","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/MJWherry/SQLScriptGenerator"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"No Default Save (#2)"}],"action":{"name":"View Issue","url":"https://github.com/MJWherry/SQLScriptGenerator/issues/2"}}}
